### DISCUSSION TITLE
Navigation Item Page Not Found Error

### DISCUSSION THREAD

**Discussion by 420skywalker420** [serenity-is/Serenity#5651] [Help] [answered]
Hi guys,

I created a new view (.cshtml file) and I want to access to that page from main menu. I added my page in NavigationItems.cs file and now i can see a link in main menu, but everytime i click on it, i get error page not found. I dont know whats problem, i think it could be the folder where i put my page, or my url in NavigationLink. It looks something likte this and its in share folder.

[assembly: NavigationLink(5008, "Privacy Policy", url: "~/PrivacyPolicy", permission: "", icon: "fa-tachometer")]

**Comment by bigjezza** [comment, marked as answer]
you need to put a controller in the folder with your CSHTML
something like this

```
namespace YourProject.PrivacyPolicy.Pages
{
    using Serenity;
    using Serenity.Web;
    using Microsoft.AspNetCore.Mvc;

    [PageAuthorize("Administration:General")]
    public class PrivacyController : Controller
    {
        [Route("~/PrivacyPolicy")]
        public ActionResult Index()
        {
            return View("~/Modules/PrivacyPolicy/PrivacyIndex.cshtml");
        }
    }
}
```

**Reply by 420skywalker420** [reply]
thank you!
